api/src/main/java/org/openmrs/DrugOrder.java

```diff
@@ -10,52 +10,88 @@
 package org.openmrs;
 
 import org.apache.commons.lang.StringUtils;
-import static org.openmrs.Order.Action.DISCONTINUE;
 import org.openmrs.util.OpenmrsUtil;
 
+import javax.persistence.JoinColumn;
+import javax.persistence.ManyToOne;
+import javax.persistence.PrimaryKeyJoinColumn;
+import javax.persistence.Entity;
+import javax.persistence.Table;
+import javax.persistence.Column;
+
+import static org.openmrs.Order.Action.DISCONTINUE;
+
 /**
  * DrugOrder
  *
  * @version 1.0
  */
+@Entity
+@Table(name = "drug_order")
+@PrimaryKeyJoinColumn(name="order_id")
 public class DrugOrder extends Order implements java.io.Serializable {
 
 	public static final long serialVersionUID = 72232L;
 
 	// Fields
 
+	// TODO - Order.hbm.xml says length is 22 but length is only used for strings. Shouldnt we use precision?
```

Comment:
I think length was an error resulting from a copy and paste of a string type element. So it can be removed.